admin_guide/scheduling/pod_placement.adoc

```diff
@@ -0,0 +1,380 @@
+from using node selectors to schedule pods.
+
+The `nodeSelectorLabelBlacklist` field of a master configuration file gives you
+control over the labels that certain roles can specify in a pod configuration's
+`nodeSelector` field. Users, service accounts, and groups that have the
+*pods/binding* permission xref:../../admin_guide/manage_rbac.adoc#admin-guide-manage-rbac[role]
+ can specify any node selector. Those without the
+*pods/binding* permission are prohibited from setting a `nodeSelector` for any
+label that appears in `nodeSelectorLabelBlacklist`.
+
+For example, an {product-title} cluster might consist of five data
+centers spread across two regions. In the U.S., "us-east", "us-central", and
+"us-west"; and in the Asia-Pacific region (APAC), "apac-east" and "apac-west".
+Each node in each geographical region is labeled accordingly. For example,
+`region: us-east`.
+
+[NOTE]
+====
+ifdef::openshift-enterprise,openshift-origin[]
+See xref:../../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[Updating
+Labels on Nodes] for details on assigning labels.
+endif::openshift-enterprise,openshift-origin[]
+ifdef::openshift-dedicated[]
+(request changes by opening a support case on the
+https://access.redhat.com/support/[Red Hat Customer Portal])
+endif::openshift-dedicated[]
+====
+
+As a cluster administrator, you can create an infrastructure where application
+developers should be deploying pods only onto the nodes closest to their
+geographical location. You can create a node selector, grouping the U.S. data centers into `superregion: us` and the APAC
+data centers into `superregion: apac`.
+
+To maintain an even loading of resources per data center, you can add the
+desired `region` to the `nodeSelectorLabelBlacklist` section of a master
+configuration. Then, whenever a developer located in the U.S. creates a pod, it
+is deployed onto a node in one of the regions with the `superregion: us` label.
+If the developer tries to target a specific region for their pod (for example,
+`region: us-east`), they receive an error. If they try again, without the
+node selector on their pod, it can still be deployed onto the region they tried
+to target, because `superregion: us` is set as the project-level node selector,
+and nodes labeled `region: us-east` are also labeled `superregion: us`.
+
+
+. Ensure you have the desired labels
+ifdef::openshift-enterprise,openshift-origin[]
+(see xref:../../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[Updating
+Labels on Nodes] for details)
+endif::openshift-enterprise,openshift-origin[]
+ifdef::openshift-dedicated[]
+(request changes by opening a support case on the
+https://access.redhat.com/support/[Red Hat Customer Portal])
+endif::openshift-dedicated[]
+and xref:../../admin_guide/managing_projects.adoc#using-node-selectors[node selector]
+set up in your environment.
+//tag::node-selectors[]
```

Comment:
@mburke5678 - because the tag moved from scheduling.adoc to this new file, any references to it needed to be moved as well. There was one reference in node_selector.adoc which was not moved.

PS: I am fixing the issues with this PR, this is just an FYI so you don't need to do anything.